doc/development/release_distribution.rst

```diff
@@ -15,137 +15,149 @@ Preamble
+   - Edit ``setup.cfg`` by changing the "Development Status" line in the ``classifiers`` section to ::
 
-Next edit ``setup.cfg``.
-Change the "Development Status" line in the ``classifiers`` section to ::
+        Development Status :: 5 - Production/Stable
 
-    Development Status :: 5 - Production/Stable
+   Commit both changes (``git add VERSION setup.cfg; git commit -m "Set release mode for 4.7.0"``), and then push them to your fork (``git push -u origin prepare-qutip-4.7.0``)
+#. Using GitHub, make a pull request to the release branch (e.g. ``qutip-4.7.X``) using this branch that you just created.
+   You will need to change the "base branch" in the pull request, because GitHub will always try to make the PR against ``master`` at first.
+   When the tests have passed, merge this in.
+#. Finally, back on ``master``, make a new pull request that changes the ``VERSION`` file to be ``<next-expected-version>.dev``, for example ``4.8.0.dev``.
+   The "Development Status" in ``setup.cfg`` on ``master`` should not have changed, and should be ::
 
-Commit both changes, and then push them to your fork ::
+       Development Status :: 2 - Pre-Alpha
 
-    $ git push --set-upstream origin 4.1-release_ready
+   because ``master`` is never directly released.
 
-Make a Pull Request to the release branch.
+You should now have a branch that you can see on the GitHub website that is called ``qutip-4.7.X`` (or whatever minor version), and the state of the code in it should be exactly what you want to release as the new minor release.
+If you notice you have made a mistake, you can make additional pull requests to the release branch to fix it.
+``master`` should look pretty similar, except the ``VERSION`` will be higher and have a ``.dev`` suffix, and the "Development Status" in ``setup.cfg`` will be different.
 
-The "Development Status" of ``master`` should remain ::
+You are now ready to actually perform the release.
+Go to deploy_. 
 
-    Development Status :: 2 - Pre-Alpha
 
-because it is never directly released.
-The ``VERSION`` file on ``master`` should reflect the last major or minor release, with a trailing ``.dev`` to indicate that this is a development branch.
 
+.. _bugfix:
 
-.. _docbuild:
-
-Documentation build
-+++++++++++++++++++
-
-Documentation should be rebuilt for a minor or major release.
-If there have been any documentation updates as part of a micro release, then it should also be built for this.
-The documentation repository is now inside ``qutip/qutip``, in the ``doc`` directory..
-
-Ensure that the following steps are complete:
-
-- The version should be changed in ``conf.py``.
-- Update ``api_doc/classes.rst`` for any new / deleted classes.
-- Update ``api_doc/functions.rst`` for any new / deleted functions.
-- Update ``changelog.rst`` including all changes that are going into the new release.
+Create a bug fix release
```

Comment:
And here we have a lower case section heading, proving your point @jakelishman. :see_no_evil:

Reply:
Yep, I'm not even consistent with myself...  I think a very small majority of the section headings in the guide are in title case (capitalise most words), so let's go with that?